README.md

```diff
@@ -244,3 +244,6 @@ docker push ${TAG}
 ```
 
 Instructions on how to run the Import Adapter Docker image locally are available on the [wiki](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/wiki/Running-Docker-image-locally).
+
+### Data protection toolkit deployment 
+The adapters can be deployed as part of the [data protection toolkit](https://github.com/GoogleCloudPlatform/healthcare/tree/master/deploy) as gke_workload resource. Sample configuration may be found in this [folder.](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/tree/master/samples) 
```

Comment:
Phrasing suggestion: "The adapters can be deployed as a gke_workload using the [data protection toolkit.](...)"

Reply:
done